tests/unit/via/views/view_video_test.py

```diff
@@ -23,11 +23,15 @@ def test_it(
         video_url,
         ViaSecurityPolicy,
     ):
+        youtube_service.canonical_video_url.return_value = sentinel.canonical_video_url
```

Comment:
You probably don't need to do this, you can just refer to `youtube_service.canonical_video_url.return_value` in the code below. Mocks already return unique values out of the box so unless the sentinel name is really adding anything to the code it's unnecessary

Reply:
I followed the pattern of the existing code here (see `sentinel.youtube_video_id` below), although it did look non-idiomatic to me. Is there a reason why the video ID has to be done that way that I've overlooked?

Reply:
Yeah, just try removing it and see what breaks, it's [set to `None` by default](https://github.com/hypothesis/via/blob/a754bdef4f83ab9f924b991209fb19849a1e0962/tests/unit/services.py#L67). It probably shouldn't be set to `None` by default though as that's a bit surprising/unexpected: whatever tests need it to be `None` should probably set that in those particular tests

Reply:
> Yeah, just try removing it and see what breaks,

I did this. It looks like this was added to the mocks so that the default behavior of any tests using it are that a URL is not a YouTube URL. Without this, a bunch of other tests started failing and would have needed to set up an override.

I have removed just the sentinel added in this test, and added a note about why the other one still exists.